### ISSUE TITLE
create layperson-only release

### ISSUE THREAD

**Issue by mellybelly** [obophenotype/human-phenotype-ontology#2490]
Numerous people are now asking for a layperson view of the HPO.
Can we consider releasing alongside the primary release file, a secondary file that has layperson synonyms as primary labels and contains closure over only those terms with lay synonyms? 

At a later date we can consider the use of lay-definitions for patient friendly use, pruning annotations, etc.

**Comment by pnrobinson** [comment]
Hi everybody,

I would volunteer to make a pipeline to do this for the obo file using HPOAPI  if this would work well with the rest of the pipeline. If somebody has a more suitable way of doing this please shout though.

-Peter


Peter Robinson

Professor of Computational Biology

The Jackson Laboratory for Genomic Medicine

10 Discovery Drive

Farmington, CT 06032

860.837.2095 t | 860.990.3130 m

peter.robinson@jax.org<mailto:peter.robinson@jax.org>

www.jax.org

Robinson lab: https://robinsongroup.github.io/

The Jackson Laboratory: Leading the search for tomorrow's cures

________________________________

**Comment by drseb** [comment]
I am not sure, that generating such an ontology is a really good idea. Especially removing terms with lay-synonyms sounds problematic. 
Also note, that once you make a file part of the release, users expect it to be maintained etc...

**Comment by cmungall** [comment]
Who is consuming this? Would they really be best served by an obo file in the first place? Although we will likely need to support obo for at least another decade, let's avoid adding new products we need to support. And also the whole needing to switch the label out thing is largely an obo artefact in the first place.

Also I can guarantee you that once you release this obo file, many people will mistake it for the real HPO, it will propagate, confuse people and make us look bad...

Maybe keep it simple: 

 - a 2 column TSV with ID, layterm
 - a 2 column TSV with child ID, to nearest lay parent ID (i.e the reduced subset graph)
 - (later) ID to lay definition

This avoids my the myriad problems of releasing a confusing 2nd obo version of the HPO. It can be generated on each release with a SPARQL query (this strategy for releasing derived TSVs/CSVs with each release is now very standard and doesn't require any custom code, and lessens @drseb's concerns about support downstream). It will clearly be a distinct report structure, and not something that would be easily confused with the main HPO release.

We could also do a json file. But really anyone who is more sophisticated than the TSV consumers should be perfectly capable of extracting the subset they require dynamically from our json.

**Comment by drseb** [comment]
Sounds reasonable @cmungall 

@mellybelly , maybe you can give us more context regarding your request?

**Comment by pnrobinson** [comment]
Hi Seb,

don't understand -- what is problematic? It is pretty simple to generate automatically and it would probably be a lot easier for users if the LP label was in the primary label--then, most software for displaying OBO files would just do the right thing. And we would not remove terms with LP synonyms, but the opposite!

-peter


Peter Robinson

Professor of Computational Biology

The Jackson Laboratory for Genomic Medicine

10 Discovery Drive

Farmington, CT 06032

860.837.2095 t | 860.990.3130 m

peter.robinson@jax.org<mailto:peter.robinson@jax.org>

www.jax.org

Robinson lab: https://robinsongroup.github.io/

The Jackson Laboratory: Leading the search for tomorrow's cures

________________________________

**Comment by drseb** [comment]
> It is pretty simple to generate automatically and it would probably be a lot easier for users if the LP label was in the primary label--then, most software for displaying OBO files would just do the right thing. 

This is not about this being simple or not. This is about being a strategically good thing. I am not convinced at this point.

> And we would not remove terms with LP synonyms, but the opposite!

I fixed my comment. Sorry for the confusion.

**Comment by pnrobinson** [comment]
@drseb @cmungall @mellybelly  Now that the LP paper is out, maybe we should revisit this? Or maybe we can just wait and see if we get requests for a special file. In principle anyone who is making software should be able to filter on the LP synonyms, and so we shouldnt prematurely optimize this?

**Comment by pnrobinson** [comment]
I think that we can close this issue since there is no clear action item. I would propose it is better to supply people with code to filter the entire HPO for LP synonyms rather than maintaining a separate file, following Seb's suggestion.
